headlamp-k8s/headlamp
# frontend: Details page blinking fixes

After #2627 was merged I've noticed some blinking that was happening when navigating to details page from the list page

This PR adds 2 fixes, one for AuthVisible query key and another for the simple table

How to test:
Navigate to Pods
Click on any Pod
No blinking/layout shifting should occur


AuthVisible fix adjust how the key for the query is defined. Previously it was passing the whole object into the key which wasn't very deterministic and would request auth again sometimes even if the object is the same.

The SimpleTable fix is described in the commit message
